magento/pwa-studio
# [Docs] component props tutorial

## Description

Revises and publishes the props tutorial from @rossmc 

## Related Issue

Closes #303

## Acceptance 

Any developer

### Verification Stakeholders

Any developer

### Specification



### Verification Steps


1. Navigate to the docs project: `cd pwa-devdocs`
2. Run linter: `yarn lint src/tutorials/pwa-studio-fundamentals/`
3. Verify no linting issues
4. Start the HTML preview server: `yarn develop`
5. Navigate to the tutorials section
6. Verify left nav entry of topic and when clicked leads to correct page

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
